java/src/com/ibm/streamsx/rest/StreamingAnalyticsConnectionInterface.java

```diff
@@ -0,0 +1,30 @@
+/*
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2017
+ */
+package com.ibm.streamsx.rest;
+
+import java.io.IOException;
+
+/**
+ * The initial interface for interacting with the REST API of a Streams instance
+ * on the Streaming Analytics Service on IBM Bluemix. Since Streaming Analytics
+ * Service has a single Streams instance (at least for now), this extends
+ * {@link StreamsConnectionInterface} and adds a convenience method {@link getInstance}
+ * to get the single instance.
+ * <p>
+ * Concrete instances are created with the factory methods in
+ * {@link StreamsRestFactory}
+ */
+public interface StreamingAnalyticsConnectionInterface extends StreamsConnectionInterface {
```

Comment:
Should we use the eclipse style here? `IStreamingAnalyticsConnection` instead of `StreamingAnalyticsConnectionInterface`.